src/ctap/ctap1.rs

```diff
@@ -282,7 +227,7 @@ impl Ctap1Command {
             U2fCommand::Version => Ok(Vec::<u8>::from(super::U2F_VERSION_STRING)),
 
             // TODO: should we return an error instead such as SW_INS_NOT_SUPPORTED?
-            U2fCommand::VendorSpecific { .. } => Err(Ctap1StatusCode::SW_NO_ERROR),
+            U2fCommand::VendorSpecific { .. } => Err(Ctap1StatusCode::SW_INTERNAL_EXCEPTION),
```

Comment:
This case is for vendor specific commands. The functionally equivalent replacement would be `SW_SUCCESS`, right? As the comment suggests, I'd also support `SW_INS_INVALID` without any expected interoperability issues. Can you explain your rational behind `SW_INTERNAL_EXCEPTION`?

Reply:
updated